src/Action/EntrypointAction.php

```diff
@@ -9,12 +9,12 @@
  * file that was distributed with this source code.
  */
 
-namespace ApiPlatform\Core\Hydra\Action;
+namespace ApiPlatform\Core\Action;
```

Comment:
I understand as is, an entrypoint is the /, the base of the app. The first thing you will see (even if there is authentification, you will be redirect or not allowed).